posthog/queries/query_date_range.py

```diff
@@ -73,7 +73,6 @@ def date_to_param(self) -> datetime:
     def get_earliest_timestamp(self):
         return get_earliest_timestamp(self._team.pk)
 
-    @cached_property
```

Comment:
Something stale got cached and the code below never ran when this was a cached property. Since we're yeeting the entire thing soon enough, I hope this is "meh, ok".

Reply:
Yeah had a similar issue a while ago somewhere... simple "computed" properties should never be cached, as it can hide bugs or introduce uncertainty.

Reply:
Yeah, that makes sense. Changed to just `@property`